### DISCUSSION TITLE
Garage Door Support

### DISCUSSION THREAD

**Discussion by Natalie-321** [iklein99/homebridge-smartthings#95] [Q&A] [answered]
Hi and thanks for this great plugin!
When I set it up, the two smart plugs I have in SmartThings appeared right away, but my Garage Door did not. I have a Nexx Smart Wi-Fi Garage Door Controller NXG-200, if that helps. Let me know if this device is supported or if there is something I missed during setup. Thanks!!

**Comment by iklein99** [comment]
I'd have to see what Smartthings is reporting about that device.  If it support the doorControl capability, it should work.  If you truncate your log, then put Homebridge into debug. mode, then restart.  Wait about 30 seconds, then download the logs and send to me.  I'll see what the door looks like.

You should then uncheck debug and restart.

You can send your logs directly to me via email.  Don't post here.  

iklein99@comcast.net

**Reply by Natalie-321** [reply]
Thanks! Email sent!

**Comment by iklein99** [comment, marked as answer]
The issue is that the device is reporting a deprecated capability of garageDoorControl rather than the current production capability of doorControl.  Do you know if there is perhaps an update for that controller?

I can try to make a change to handle, but I can't guarantee that it will work.

**Reply by Natalie-321** [reply]
No firmware updates that I can see in the Nexx app for my garage door controller. Do you think it's worth reaching out to the Nexx support team and asking them to replace the garageDoorControl capability with doorControl?

Or if you think it's worth making a change to the plugin to work for the deprecated capability, I'd be willing to try that too. Thanks again!

**Reply by Natalie-321** [reply]
I'm waiting for a manager to get back to me from Nexx support about whether or not they plan on updating their SmartThings integration for my garage door controller. 

In the mean time, if it's an easy change to make to try to handle the deprecated capability, it would be much appreciated and I will be happy to test it out. Have a great weekend!
